src/uct/ib/ud/base/ud_iface.c

```diff
@@ -30,7 +30,9 @@ static ucs_stats_class_t uct_ud_iface_stats_class = {
     .num_counters  = UCT_UD_IFACE_STAT_LAST,
     .class_id      = UCS_STATS_CLASS_ID_INVALID,
     .counter_names = {
-        [UCT_UD_IFACE_STAT_RX_DROP] = "rx_drop"
+        [UCT_UD_IFACE_STAT_RX_DROP] = "rx_drop",
+        [UCT_UD_IFACE_STAT_RX_COMPLETION] = "rx_completion",
+        [UCT_UD_IFACE_STAT_TX_COMPLETION] = "tx_completion",
```

Comment:
can we declare it inside `uct_ib_iface_stats_class` and remove rx/tx completion statistic from RC iface statistics? since it is common for RC/DC and UD transports now.